change/@ni-nimble-angular-d3cc8e4d-0cf0-4779-bb62-e8e09f599d10.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
There's a note on the linked issue about adding docs for these tag variables. That doesn't have to be part of this PR but if it isn't then please update the issue to ensure the remaining work is still accurately tracked.

If you do want to document it now, I don't have a strong opinion about what that should look like. Perhaps a note in the README for `nimble-components` and/or `nimble-angular` that describes the use case (getting the element tag name in a compile-time dependency / type safe way) and the preferred approach (using the exported value) and the not-recommended approach (`DesignSystem.tagFor()`)?